docs/changelog/105391.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 105391
+summary: Move `MlConfigVersion.getMinMlConfigVersion` call under try-catch block
```

Comment:
The release note should be something that’s meaningful to an end user. What was the problem that happened before that won’t happen now?

Reply:
Well... there is no problem currently. 
This change is just a safety measure if there is an exception thrown by  `MlConfigVersion.getMinMlConfigVersion` that bubbles up to the ingest processor code.

I've just changed the release note to reflect that.